tasks/php_apache.yml

```diff
@@ -1,5 +1,5 @@
   - name: LAMP | FAST-CGI configuration for php
-    template: src={{role_dir}}/templates/apache2/conf/fastcgi.conf.j2 dest=/etc/apache2/conf-available/fastcgi.conf
+    template: src={{role_dir}}/templates/apache2/conf/fastcgi-{{ php_family_prefix }}.conf.j2 dest=/etc/apache2/conf-available/fastcgi.conf
```

Comment:
Somehow did not have setup with php7 yet.  For php5 packages where called php5-*
Did they change approach with php7 to be just php-* ?